.cookiecutter.json

```
@@ -21,7 +21,7 @@
         "_drift_manager": {
             "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
             "template_dir": "nautobot-app",
-            "template_ref": "refs/tags/nautobot-app-v2.4.2",
+            "template_ref": "nautobot-app-v2.5.0",
```

It looks like the manual run is removing the "refs/tags/". For consistency, we should. add this back in.
```suggestion
            "template_ref": "refs/tags/nautobot-app-v2.5.0",
```